pages/index.js

```diff
@@ -1,3 +1,28 @@
+import styled from "styled-components";
+import SpotCard from "../components/SpotCard";
+import spots from "../helpers/mockSpots";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  return (
+    <>
+      <Headline>ParentHood Spots</Headline>
+
+      <SpotList>
+        {spots
+          ?.sort((a, b) => a.name.localeCompare(b.name))
+          ?.map((spot) => (
+            <SpotCard spot={spot} key={spot.name}></SpotCard>
+          ))}
+      </SpotList>
+    </>
+  );
 }
+
+const Headline = styled.h1`
+  text-align: center;
+`;
+
+const SpotList = styled.ul`
+  list-style: none;
+  max-width: 500px;
+`;
```

Comment:
👇This one belongs to the comment in `SpotCard.js` regarding the negative `margin value`

`<ul>` elements have a default padding-left of 40px. If you want to get rid of the cringe left padding, set the `padding` to `0`
```suggestion
const SpotList = styled.ul`
  list-style: none;
  max-width: 500px;
  padding: 0;
`;
```